docs/build-notifications.rst

```diff
@@ -20,31 +20,256 @@ Take these steps to enable build notifications using email:
 
 You should now get notified by email when your builds fail!
 
-Using webhook
--------------
+Build Status Webhooks
+------------------------------
 
 Read the Docs can also send webhooks when builds are triggered, successful or failed.
 
 Take these steps to enable build notifications using a webhook:
 
-* Go to :guilabel:`Admin` > :guilabel:`Notifications` in your project.
-* Fill in the **URL** field under the **New Webhook Notifications** heading
-* Submit the form
+* Go to :guilabel:`Admin` > :guilabel:`Webhooks` in your project.
+* Fill in the **URL** field and select what events will trigger the webhook
+* Modify the payload or leave the default (see below)
+* Click on :guilabel:`Save`
+
+.. figure:: /_static/images/webhooks-events.png
+   :align: center
+   :alt: URL and events for a webhook
+
+   URL and events for a webhook
+
+Every time one of the checked events triggers,
+Read the Docs will send a POST request to your webhook URL.
+The default payload will look like this:
+
+.. code-block:: json
+
+   {
+       "event": "build:triggered",
+       "name": "docs",
+       "slug": "docs",
+       "version": "latest",
+       "commit": "2552bb609ca46865dc36401dee0b1865a0aee52d",
+       "build": 15173336,
+       "start_date": "2021-11-03T16:23:14",
+       "build_url": "https://readthedocs.org/projects/docs/builds/15173336/",
+       "docs_url": "http://docs.readthedocs.io/en/latest/"
+   }
+
+When a webhook is sent, a new entry will be added to the
+"Recent Activity" table. By clicking on each individual entry,
+you will see the server response, the webhook request, and the payload.
+
+.. figure:: /_static/images/webhooks-activity.png
+   :align: center
+   :alt: Activity of a webhook
+
+   Activity of a webhook
+
+Custom payload examples
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can customize the payload of the webhook to suit your needs,
+as long as it is valid JSON. Below you have a couple of examples,
+and in the following section you will find all the available variables.
+
+.. figure:: /_static/images/webhooks-payload.png
+   :width: 80%
+   :align: center
+   :alt: Custom payload
 
-The project name, slug and its details for the build will be sent as POST request to your webhook URL:
+   Custom payload
+
+Slack
++++++
+
+.. code-block:: json
+
+   {
+     "attachments": [
+       {
+         "color": "#db3238",
+         "blocks": [
+           {
+             "type": "section",
+             "text": {
+               "type": "mrkdwn",
+               "text": "*Read the Docs build failed*"
+             }
+           },
+           {
+             "type": "section",
+             "fields": [
+               {
+                 "type": "mrkdwn",
+                 "text": "*Project*: <{{project.url}}|{{project.name}}>"
```

Comment:
I think it's more common the version with one space, but the zero spaces is allowed too (but I think we should document just one version).

```suggestion
                 "text": "*Project*: <{{ project.url }}|{{ project.name }}>"
```